cookies_and_ovens.js

```diff
@@ -8,3 +8,32 @@
 //
 //
 // Your code here
+
+const Kue = require('./Kue')
+
+class Coklat extends Kue{
+    constructor(){
+        super()
+        this._done = 20
```

Comment:
kalo di semuanya punya this._done, mending di set di property parentnya, nanti di anaknya tinggal kirim via super